#
# From http://sources.gentoo.org/viewcvs.py/gentoo-x86/media-libs/flac/files/flac-1.2.1-gcc-4.3-includes.patch?rev=1.1&view=markup
#
Sun Nov 18 19:31:35 2007 UTC (4 months ago) by aballier
Branch: MAIN
CVS Tags: HEAD

fix building with gcc 4.3, thanks to Ryan Hill <dirtyepic@gentoo.org>, bug #199579
(Portage version: 2.1.3.19)

diff -Naur flac-1.2.1-orig/examples/cpp/encode/file/main.cpp flac-1.2.1/examples/cpp/encode/file/main.cpp
--- flac-1.2.1-orig/examples/cpp/encode/file/main.cpp	2007-09-13 09:58:03.000000000 -0600
+++ flac-1.2.1/examples/cpp/encode/file/main.cpp	2007-11-18 12:59:45.000000000 -0600
@@ -30,6 +30,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <cstring>
 #include "FLAC++/metadata.h"
 #include "FLAC++/encoder.h"
 
